src/test/java/racing/RacingServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package racing;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import util.RacingValidator;
+import util.RandomNumberGenerator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingServiceTest {
+
+    @Test
+    @DisplayName("자동차가 1칸 앞으로 전진한다.")
+    void 자동차_전진_테스트() {
+        Car car = new Car(0);
+        RacingService.moveForward(car);
+        assertThat(car.getLocation()).isEqualTo(1);
+    }
+
+    @Test
+    @DisplayName("0에서 9 사이의 랜덤 숫자를 얻는다.")
+    void 랜덤_숫자_얻기_테스트() {
+        Integer randomNumber = RandomNumberGenerator.getRandomNumber();
+        assertThat(isNumberInRange(randomNumber)).isTrue();
+    }
+
+    @Test
+    @DisplayName("전진할 수 있는지 여부를 확인한다.")
+    void 전진_가능_여부_테스트() {
+        Integer randomNumber = RandomNumberGenerator.getRandomNumber();
+        if (randomNumber >= 4) {
+            assertThat(RacingValidator.isMovable(randomNumber)).isTrue();
+            return;
+        }
```

Comment:
```suggestion
        if (randomNumber >= 4) {
            assertThat(RacingValidator.isMovable(randomNumber)).isTrue();
            return;
        }
```
테스트 코드에 비즈니스 로직이 있네요.
테스트 코드에 비즈니스 로직이 있으면, 프로덕션 코드가 변경시 테스트 코드또한 항상 같이 수정해줘야하기 때문에 오히려 테스트 코드가 유지보수와 확장성을 방해하는 요소가 될수 있다고 생각되어요.

정말 검증하고자 하는 메소드에 집중해보아요.
```
assertThat(RacingValidator.isMovable(5)).isTrue();
```

Reply:
아 그렇겠네요 비즈니스 로직이 테스트코드에 들어가면 안되겠군요...ㅎㅎ 수정하겠습니다!